### ISSUE TITLE
Renataki's Soul Charm,Trinket Bug,Medium Priority,Item Spell

### ISSUE THREAD

**Issue by masterkss** [dobidoberman1/Mystic-5.4.8-Bug-Tracker#160]
Renataki's Soul Charm trinket gives only ~807 agility per proc, doesnt matter if it's a normal, upgraded or thunderforged version (i have all 3), instead of 1333 agility on normal and 1410 agility on thunderforged.

**Comment by baby7703jane** [comment]
It is the same with the trinkett you buy from shado pan. "Blossom of pure snow" on use it should give you 3595 Crittical strike, but you only get 86

**Comment by masterkss** [comment]
Nope. Its not fixed. It still gives 807 agility per proc. Just checked it on all 3 versions of it. Please fix it again dobidoberman.
